pkg/apis/events/v1alpha1/cloudauditlogssource_validation_test.go

```diff
@@ -57,6 +56,29 @@ var (
 			Project: "my-eventing-project",
 		},
 	}
+
+	auditLogsSourceSpecWithKSA = CloudAuditLogsSourceSpec{
+		ServiceName:  "foo",
```

Comment:
Can we use constants here for all similar tests to avoid magic string for each sources?